jlm/hls/backend/rhls2firrtl/verilator-harness-hls.hpp

```diff
@@ -23,7 +23,41 @@ class VerilatorHarnessHLS : public BaseHLS
   std::string
   get_text(llvm::RvsdgModule & rm) override;
 
+public:
+
+  /**
+  * Construct a Verilator harness generator.
+  *
+  * /param verilogFile The filepath to the input Verilog file to Verilator
+  */
+  VerilatorHarnessHLS(const util::filepath verilogFile)
+    : VerilogFile_(verilogFile) {};
+
 private:
+  const util::filepath VerilogFile_;
+
+  /**
+   * Get the include name.
+   *
+   * \return The include name.
+   */
+  [[nodiscard]] const std::string
+  GetIncludeName() const noexcept
+  {
+    return VerilogFile_.base() + ".h";
+  }
+
+  /**
+   * Get the V_NAME.
+   *
+   * \return The V_NAME.
+   */
```

Comment:
The documentation is useless. It just repeats the method name. What is the VName? What is it used for?